packages/quantic/force-app/main/default/lwc/quanticDidYouMean/quanticDidYouMean.js

```diff
@@ -0,0 +1,70 @@
+import { api, LightningElement, track } from 'lwc';
+import { initializeWithHeadless, registerComponentForInit } from 'c/quanticHeadlessLoader';
+
+export default class QuanticDidYouMean extends LightningElement {
+    @track state = {};
```

Comment:
`state` does not seem to be used.

Reply:
Yea if you explode the state like you are doing in the updateState (which is the right way to do it I think), you don't need to track state.